software/composio.yml

```
@@ -0,0 +1,11 @@
+name: Composio
+website_url: https://composio.dev/
+source_code_url: https://github.com/ComposioHQ/composio
+description: Integration platform for AI agents & LLMs with support for 250+ apps and custom tools.
+licenses:
+  - Unlicense
```

https://github.com/ComposioHQ/composio/blob/master/LICENSE

```suggestion
  - Elastic-2.0
```